test/VaultSharp.Samples/Program.cs

```diff
@@ -103,7 +103,7 @@ private static VaultClientSettings GetVaultClientSettings(IAuthMethodInfo authMe
                         Console.WriteLine(_responseContent);
                     }
                 },
-                Namespace = "bhjk",
+                //Namespace = "bhjk", // ?? 
```

Comment:
I'm not sure why the namespace was in there, but it was causing my runs to fail until I commented it out.

Reply:
namespaces is a feature available on enterprise versions of Vault. This is a placeholder I reckon. Uncomment before this is merged.